.github/workflows/osv-scanner.yml

```diff
@@ -14,36 +11,26 @@ on:
   - cron: '44 15 * * 5'
 
 permissions:
+  actions: read
   contents: read
+  # Require writing security events to upload SARIF file to security tab
+  security-events: write
```

Comment:
Declaring write permission on workflow top level will reduce Token-Permissions score in ossf-scorecard. You should declare `contents: read` on top level and full permissions on each job level.

Reply:
<img width="786" alt="image" src="https://github.com/user-attachments/assets/dd11d98c-dd46-4788-901f-b79d428c2147" />
follow up the offical doc, is that not right?